deploy/examples/create-external-cluster-resources-tests.py

```diff
@@ -11,6 +11,8 @@
 limitations under the License.
 """
 
+# pylint: disable=E0606
```

Comment:
I see that the `target = None` change is present here (https://github.com/rook/rook/pull/14204/files) but not in this PR. Would it make more sense to use `cherry-pick -x` to include that commit here instead of copying only one part of its change?

Reply:
I dont think so that is related to the CI